js/models/conversations.js

```diff
@@ -2241,7 +2241,7 @@
       if (this.isPrivate() && !this.get('name')) {
         return this.get('profileName');
       }
-      return null;
+      return this.getTitle();
```

Comment:
Oops i didn't catch this, but this could potentially create an infinite loop.
Simplest method would be to do the following:
```suggestion
      return this.getName();
```

Reply:
Ah derp ofc
I'm not 100% what `getName` will return for groups :thinking: might have to revisit how signal is doing the group names, I inspected the signal desktop and it's quite a bit different